node/node.go

```diff
@@ -180,7 +180,7 @@ func (n *Node) Start(ctx context.Context) error {
 			return fmt.Errorf("failed to register the operator: %w", err)
 		}
 	} else {
-		n.Logger.Info("The node has successfully started. Note it's not opt-in to EigenDA yet (it's not receiving or validating data in EigenDA). To register, please follow the EigenDA operator guide section in docs.eigenlayer.xyz")
+		n.Logger.Info("The node has successfully started. Note: if it's not opted in to EigenDA (it's not receiving or validating data in EigenDA) then please follow the EigenDA operator guide section in docs.eigenlayer.xyz to register")
```

Comment:
For the "Note" part, how about we ask operators to check if they have opted in on the https://goerli.eigenlayer.xyz/avs/eigenda?

The message here will be like:
"The node has successfully started. Note: please check if you have opted in on https://goerli.eigenlayer.xyz/avs/eigenda. If not, please follow the EigenDA operator guide ...."

It'll be then clear and actionable in each step.

Reply:
I think that's a v good idea. I will update.

Reply:
One caveat: it's tied to Goerli, which will need to be updated. Or we may maintain a list (if we have other chain), like "for Goerli, check xxx; for mainet, check xxxx" etc.

Reply:
yea I was looking for if we have chainID, but we don't. eventually would be cool to figure out chainID and have the correct URL. I will keep it right now and mention goerli